source/hands_on/trex.rst

```diff
@@ -0,0 +1,82 @@
+TREX
+^^^^^^^
+
+Overview
+---------
+TREX (Trajectory-ranked Reward Extrapolation) was first proposed in `Extrapolating Beyond Suboptimal Demonstrations via Inverse Reinforcement Learning from Observations <https://arxiv.org/abs/1904.06387>`_, which uses ranked demonstrations to inference reward function. Different from previous methods, TREX seeks a reward function that explains the ranking over demonstrations, rather than justifying the demonstrations. Hence, this algorithm can learn good policies from highly suboptimal demostrations.
+
+Quick Facts

```

Comment:
On key idea, which also indicated on the title of the paper, is the extrapolation beyond the suboptimal demonstrations data. Would you please also emphasise this key point here?